packages/reports/tests/integration/components/filter-values/date-test.ts

```diff
@@ -0,0 +1,65 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { click, render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+import { TestContext as Context } from 'ember-test-helpers';
+import FilterFragment from 'navi-core/models/bard-request-v2/fragments/filter';
+import FragmentFactory from 'navi-core/addon/services/fragment-factory';
+//@ts-ignore
+import { setupMirage } from 'ember-cli-mirage/test-support';
+
+interface TestContext extends Context {
```

Comment:
Something like this makes it nicer since you don't have to duplicate the types when writing component integration tests
https://github.com/yahoo/navi/blob/ab54ef84354bc1f8086408d06ecee55912b581b0/packages/core/tests/integration/components/navi-visualizations/metric-label-test.ts#L7-L19

Reply:
👍  let come back, we need to get this in.